src/makeRobot.js

```diff
@@ -39,6 +39,60 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    get info() {
+      return `name: ${this.name}, chip version: ${
+        this.version}, wheels: ${this.wheels}`;
+    },
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goRight(value = 1) {
+      if (value < 0) {
+        return this;
+      }
+      this.coords.x += value;
+
+      return this;
+    },
```

Comment:
```suggestion
    goRight(value = 1) {
      if (value >= 0) {
        this.coords.x += value;
      }

      return this;
    },
```
Можна поміняти перевірку і трохи скоротити код.